### PR TITLE
[go-dhcp] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
The DHCP library is used by many [PublicProjects](https://github.com/insomniacslk/dhcp#public-projects-that-use-it).

Merged PR : https://github.com/insomniacslk/dhcp/pull/508

My Proposal https://github.com/insomniacslk/dhcp/pull/508#discussion_r1268886577

### CODE REVIEW

File: projects/go-dhcp/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-go
+RUN git clone --depth 1 https://github.com/0x34d/dhcp.git
```
Comment (0x34d) [inline projects/go-dhcp/Dockerfile:18]: change after this https://github.com/insomniacslk/dhcp/pull/508

File: projects/go-dhcp/project.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/insomniacslk/dhcp"
+language: go
+primary_contact: ""
```
Comment (0x34d) [inline projects/go-dhcp/project.yaml:3]: I don't have it yet, but I will get it.

File: projects/go-dhcp/project.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/insomniacslk/dhcp"
+language: go
+primary_contact: "insomniac@slackware.it"
```
Comment (0x34d) [inline projects/go-dhcp/project.yaml:3]: @pmazzini @insomniacslk is this good ?

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**